src/audio/host-zephyr.c

```diff
@@ -5,32 +5,7 @@
 // Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
 //         Keyon Jie <yang.jie@linux.intel.com>
 
-#include <sof/audio/buffer.h>
-#include <sof/audio/component_ext.h>
-#include <sof/audio/pcm_converter.h>
-#include <sof/audio/pipeline.h>
-#include <sof/audio/ipc-config.h>
-#include <sof/common.h>
-#include <sof/debug/panic.h>
-#include <sof/ipc/msg.h>
-#include <rtos/alloc.h>
-#include <sof/lib/dma.h>
-#include <sof/lib/mailbox.h>
-#include <sof/lib/memory.h>
-#include <sof/lib/notifier.h>
-#include <sof/lib/uuid.h>
-#include <sof/list.h>
-#include <sof/math/numbers.h>
-#include <rtos/string.h>
-#include <sof/ut.h>
-#include <sof/trace/trace.h>
-#include <ipc/stream.h>
-#include <ipc/topology.h>
-#include <user/trace.h>
-#include <errno.h>
-#include <stdbool.h>
-#include <stddef.h>
-#include <stdint.h>
+#include <sof/audio/host_copier.h>
```

Comment:
please, don't do this. Headers should be included explicitly

Reply:
ok, changing, hope below docker build issue also can be resolved:
https://github.com/thesofproject/sof/actions/runs/3948546707/jobs/6758664468

Reply:
that build issue is an example of why headers should be included where used. In that case the header, defining `struct dma_config` (i.e. zephyr/drivers/dma.h) should be included in host_copier.h
As for this hunk - I still don't think you should remove those headers: buffer.h is needed for `struct comp_buffer`, pcm_converter.h is needed for `pcm_converter_func`, component_ext.h might indeed be not needed, but I haven't checked the whole of host-zephyr.c

Reply:
@btian1 could you address comments, please? thanks.

Reply:
Thanks, I will add them back:
#include <sof/audio/component_ext.h>	
#include <sof/audio/pcm_converter.h>